components/select/src/auro-select.js

```diff
@@ -51,7 +51,7 @@ export class AuroSelect extends LitElement {
   constructor() {
     super();
 
-    this.placeholder = 'Please select option';
+    this.placeholder = '';
```

Comment:
Why was this changed?

Reply:
I made the change because of 3 reasons. 
1. I was told that we prefer properties to be additive.
2. placeholder is optional, not required. With that, English value as default would fail on i18n
However, I'm happy to revert back if we still want to have a default value.

Reply:
Yes we should keep this for now and this should be brought up as a conversation to the team. We do not usually populate attribute with text values, but @jason-capsule42 probably knows better why the `placeholder` already has a value.